docs/Installation/configure_ingress.rst

```diff
@@ -0,0 +1,40 @@
+=============================
+Configuring Ingress for Zenko
+=============================
+
+Zenko’s supported standard deployment uses MetalK8s with NGINX ingress
+control. You can use other ingress controllers and other Kubernetes engines
+to support Zenko, but all possible configurations of ingress controller and
+Kubernetes engine are beyond scope.
+
+If you are not using the default MetalK8s/NGINX deployment, use the following

```

Comment:
If your Kubernetes cluster has an NGINX ingress controller running, use the following guidelines to configure an HTTPS ingress resource.

Reply:
done